examples/llama_guard/standalone_inference.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the Llama 2 Community License Agreement.
+
+import fire
+import os
+import sys
+
+# Get the directory of the current script
+current_dir = os.path.dirname(os.path.abspath(__file__))
+
+# Get the parent directory
+parent_dir = os.path.dirname(current_dir)
+
+# Append the parent directory to sys.path
+sys.path.append(parent_dir)
+
+from llama_guard.generation import Llama
```

Comment:
good call on the prompt_format. Will change it, as it's pure python. On the generator, we decided to keep it in the llama_guard example folder because it's using plan [llama](https://github.com/facebookresearch/llama/blob/main/llama/generation.py) classes instead of the transformer lib. Once I update the code with that lib, I can move it to the inference folder as well.